src/views/FindOrder.vue

```diff
@@ -259,11 +259,31 @@ export default defineComponent ({
   },
   async mounted() {
     await this.getOrders();
-    await this.store.dispatch('order/getPurchaseOrderIds').then(ids => {
-      if(ids.length > 0) {
-        this.poIds = ids
+
+    const payload = {
+      "json": {
+        "params": {
+          "rows": 1000,
+          "group": true,
+          "group.field": "externalOrderId"
+        },
+        "filter": "docType: ORDER AND orderTypeId: PURCHASE_ORDER",
+        "fields": "externalOrderId orderId",
+        "query": "*:*"
       }
-    })
+    }
+
+    const resp = await OrderService.getPOIds(payload);
+    if (resp.status == 200 && !hasError(resp)) {
+      resp.data.grouped.externalOrderId.groups.map((group: any) => {
+        if (group.groupValue) {
```

Comment:
We could try with solr query that we exclude empty values

Reply:
Sir, filtered results using solr query.